### ISSUE TITLE
[Feature][jira] translate status changelog to user lang when convert issue changelogs

### ISSUE THREAD

**Issue by narrowizard** [apache/incubator-devlake#3991]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-devlake/issues?q=is%3Aissue) and found no similar feature requirement.

### Use case

We found that the changelog value of type status is always in english **language** when we using jira server.  
Now we implemented a new feature to extend std status (in issue tracking domain). The different language of status value between issues and issue changelogs makes us confused.

### Description

I recommend converting the translated value of changelog status and saving it to the original_from_value (or original_to_value) field (in issue changelog convertor).

### Related issues

#2181

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by narrowizard** [comment]
## More info about my jira server
- version: v8.16.1
- user lang: chinese 
- partial api response
```js
// /rest/api/2/status, with translated name
[
    {
        "self": "http://172.26.129.92:8092/rest/api/2/status/1",
        "description": "问题已经准备好让经办人开始处理。",
        "iconUrl": "http://172.26.129.92:8092/images/icons/statuses/open.png",
        "name": "开放",
        "id": "1",
        "statusCategory": {
            "self": "http://172.26.129.92:8092/rest/api/2/statuscategory/2",
            "id": 2,
            "key": "new",
            "colorName": "blue-gray",
            "name": "待办"
        }
    }, ...
]
// /rest/agile/1.0/board/1/issue?expand=changelog
// issue's status is with translated name
// changelogs' from_string is with untranslated name
{
    "expand": "schema,names",
    "startAt": 0,
    "maxResults": 50,
    "total": 287,
    "issues": [
        {
            "expand": "operations,versionedRepresentations,editmeta,changelog,renderedFields",
            "id": "10128",
            "self": "http://172.26.129.92:8092/rest/agile/1.0/issue/10128",
            "key": "AAA-29",
            "fields": {
                // ...
                "status": {
                    "self": "http://172.26.129.92:8092/rest/api/2/status/10001",
                    "description": "",
                    "iconUrl": "http://172.26.129.92:8092/",
                    "name": "完成",
                    "id": "10001",
                    "statusCategory": {
                        "self": "http://172.26.129.92:8092/rest/api/2/statuscategory/3",
                        "id": 3,
                        "key": "done",
                        "colorName": "green",
                        "name": "完成"
                    }
                },
                // ...
            },
            "changelog": {
                "startAt": 0,
                "maxResults": 29,
                "total": 29,
                "histories": [
                    {
                        "id": "1011010",
                        // ...
                        "items": [
                            {
                                "field": "status",
                                "fieldtype": "jira",
                                "from": "10000",
                                "fromString": "To Do",
                                "to": "10000",
                                "toString": "To Do"
                            }
                        ]
                    },
                    // ...
                ]
            }
        },
        // ...
    ]
}
```

**Comment by klesh** [comment]
1. The `issue` and `status` from Jira Server API provide only the **translated status name** along with **status id**
2. While the `changelog` offers only the **untranslated name** along with **status id**
3. And, the **status mapping mechanism** in our system relies on the **translated status name** from the `status` api for user to select.
4. The problem is, we need to unify the **status name** either in **translated** or **untranslated** form in order to track the issue cycle times.
5. However, unifying them by **untranslated name** is impossible due to api limitation listed above. It leaves us no choice but to make the `changelog.status_name` **translated**

**Comment by narrowizard** [comment]
Fixes in #2181
